cmds/webboot/webboot.go

```diff
@@ -50,6 +51,10 @@ func (i *ISO) exec(uiEvents <-chan ui.Event, boot bool) error {
 		distro = supportedDistros[entry.Label()]
 	}
 
+	verbose("Using distro %s with boot config %s", distroName, distro.bootConfig)
+
+	// Magic trick: The parsers may write very long and verbose output; too much!
+	log.SetOutput(&nilBuffer)
```

Comment:
I don't like just throwing  output away. A file in /tmp, fine. But just throwing it away makes debugging really hard.

Reply:
I'll get back to the file thing. For now I'll keep this as is.